src/deferred-binding/connectionDetails.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
We should move this file inside storage blob? Each extension should gets its own connection details. There is no common method that can be shared with all extensions.,

Reply:
This is the modelBindingData( deferred binding) that we are processing, I would think for all the types we will need a way to parse the data in buffer object.

Based on the model parameter for other deferred binding types, we may have to change the logic slightly. I will refactor when, I will see the data from service bus or cosmos.

But today I will keep it unchanged

Reply:
Also, currently model binding data is core-types I need to see what needs to be exposed. As there is no customer impact and customer is unaware of this, I will lkeep it this iteration. Later I will refactir.

Reply:
This is fixed and moved to extensions